docs/theory.rst

```diff
@@ -200,9 +200,10 @@ The usual manifestation of these roundoff errors is high-amplitude small-scale n
     In such cases, the user has a few options to try to regain stability.
 
     1. If the data being filtered is single-precision, it might help to promote it to double precision (or higher) before filtering.
-    2. The user can also try reducing ``n_steps``, but must not reduce it too much or the resulting filter will not behave as expected.
-    3. Users might elect to *coarsen* their data before filtering, i.e. to reduce the resolution of the input data before applying the filter. This has the effect of increasing the grid size, and thus decreasing the gap between the filter scale and the grid scale.
-    4. The final option is simply to use a different approach to filtering, not based on ``gcm-filters``.
+    2. If a user is encountering instability with the standard Gaussian filter, the user can try setting ``n_iterations`` to an integer greater than 1. Read more about this in :doc:`factored_gaussian`.
+    3. The user can also try reducing ``n_steps``, but must not reduce it too much or the resulting filter will not behave as expected.
```

Comment:
Can we give better guidance on what is meant by "too much"?

Reply:
This isn't really a comment about this PR; the offending snippet is in the main branch. Could we deal with this separately?